crates/fs/src/fs.rs

```diff
@@ -548,6 +548,10 @@ impl Fs for RealFs {
         let pending_paths: Arc<Mutex<Vec<PathEvent>>> = Default::default();
         let root_path = path.to_path_buf();
 
+        // Check if root path is a symlink
+        let tr_path = self.read_link(&path).await.ok();
+        let tr_path_clone = tr_path.clone();
```

Comment:
Rather than have this be `tr_path_clone`, let's use the block-clone-return style to capture the local variable, like this: https://github.com/zed-industries/zed/blob/main/crates/fs/src/fs.rs#L635-L637

Reply:
And while we're at, let's call this variable `target_path`, maybe. We prefer not to have abbreviations :)